6-data-storage/02-localstorage/1-form-autosave/task.md

```diff
@@ -3,7 +3,7 @@
 
 Create a `textarea` field that "autosaves" its value on every change.
 
-So, if the user accidentally closes the page, and opens it again, he'll find his unfinished input at place.
+So, if the user accidentally closes the page, and opens it again, they'll find his unfinished input at place.
```

Comment:
```suggestion
So, if the users accidentally close the page, then open it again, they'll find his unfinished input at place.
```

if we're going plural, we may need to switch to users also

Reply:
```suggestion
So, if the users accidentally close the page, then open it again, they'll find their unfinished input at place.
```
his can also be changed to their

Reply:
[singular they](https://en.wikipedia.org/wiki/Singular_they) works fine IMO, no need for `users`